File: rb/test/exists_test.rb
# Dingconnect SDK exists test

require "minitest/autorun"
require_relative "../Dingconnect_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = DingconnectSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: .sdk/tm/js/test/exists.test.js

const { test, describe } = require('node:test')
const { equal } = require('node:assert')


const { DingconnectSDK } = require('..')


describe('exists', async () => {

  test('test-mode', async () => {
    const testsdk = await DingconnectSDK.test()
    equal(null !== testsdk, true)
  })

})
